api/api/spec/spec.yaml

```diff
@@ -311,68 +338,76 @@ components:
       type: object
       properties:
         profile:
+          description: CIS-CAT profile scanned.
           type: string
         score:
+          description: Percentage of passed checks.
           type: integer
         error:
+          description: "Number of checks that CIS-CAT wasn't able to run."
           type: integer
         scan:
-          type: object
-          properties:
-            id:
-              type: integer
-            time:
-              type: string
-              format: date-time
+          $ref: '#/components/schemas/ScanIdTime'
         fail:
+          description: Number of failed checks. If this number is higher than 0 the host will probably have a vulnerability.
           type: integer
         benchmark:
-          type: integer
+          description: CIS-CAT benchmark where the profile is defined.
+          type: string
         pass:
+          description: Number of passed checks.
           type: integer
         notchecked:
+          description: Number of not passed checks.
           type: integer
         unknown:
+          description: Number of checks which status CIS-CAT wasn't able to determine.
           type: integer
 
     ## Cluster models
-    ClusterNode:
+    ClusterNodeBasic:
       type: object
       properties:
-        name:
-          type: string
         type:
           type: string
           enum:
           - worker
           - master
+          description: Node type.
         version:
           type: string
           format: \d+\.\d+\.\d+
+          description: Wazuh version installed in the node.
         ip:
           type: string
+          description: IP the node is using to communicate with other nodes in the cluster.
 
+
+    ClusterNode:
+      allOf:
+        - $ref: '#/components/schemas/ClusterNodeBasic'
+        - type: object
+          properties:
+            name:
```

Comment:
This `name` can be refactored.

Reply:
Done in 408a7a8d29368b3b877e7b6d1303d159d81cb1ee